core/metadata.js

```diff
@@ -1,8 +1,9 @@
 /* @flow */
 
 const _ = require('lodash')
-const { clone, isEqual, pick } = _
+const { clone } = _
 const mime = require('mime')
+const delta = require('deep-diff').diff
```

Comment:
Is the trailing `.diff` required? E.g. wouldn't `const delta = require('deep-diff')` work?

Also what about `deepDiff` instead of `delta`, to match the lib name, so we don't introduce another one?

```js
const deepDiff = require('deep-diff')
```